addons/l10n_ae/models/template_ae.py

```diff
@@ -31,3 +31,37 @@ def _get_ae_res_company(self):
                 'account_journal_early_pay_discount_gain_account_id': 'uae_account_500014',
             },
         }
+
+    @template('ae', 'account.journal')
+    def _get_ae_account_journal(self):
+        """ If UAE chart, we add 2 new journals TA and IFRS"""
+        return {
+            "tax_adjustment":{
+                "name": "Tax Adjustments",
+                "code": "TA",
+                "type": "general",
+                "show_on_dashboard": True,
+                "sequence": 1,
+            },
+            "ifrs16": {
+                "name": "IFRS 16",
+                "code": "IFRS",
+                "type": "general",
+                "show_on_dashboard": True,
+                "sequence": 10,
+            }
+        }
+
+    @template('ae', 'account.account')
+    def _get_ae_account_account(self):
+        return {
+            "uae_account_100101": {
+                'allowed_journal_ids': [Command.link('ifrs16')],
+            },
+            "uae_account_100102": {
+                'allowed_journal_ids': [Command.link('ifrs16')],
+            },
+            "uae_account_400070": {
```

Comment:
I don't like we have 2x `_inherit = 'account.chart.template'` in some localization now, sometimes with random names like this one "template_ae.py"... What do you plan to do with that? By convention, we should keep a since file `account_chart_template.py` in every l10n

Reply:
This is exactly what I'm removing with this commit...

And `template_ae.py` is **not** a random name. It is the file that contains all the `@template('ae'...` In most cases (all but one), you won't have a `account_chart_template.py` in the same folder